include/oneapi/dpl/pstl/hetero/algorithm_impl_hetero.h

```diff
@@ -1229,6 +1229,16 @@ __pattern_is_heap(_ExecutionPolicy&& __exec, _RandomAccessIterator __first, _Ran
         __par_backend_hetero::make_iter_mode<__par_backend_hetero::access_mode::read>(__last), _Predicate{__comp});
 }
 
+template <typename Name>
+class merge_n1_equals_0
```

Comment:
The goal is the same as for all such classes in the file. Comment can be the same for all such cases. It can be grouped in the separate PR to not make this one too long.

Reply:
About `merge_n1_equals_0`
It means the first sequence  is empty. 
Essentially, it is copying. Probably, we can re-use a name of the existing copying kernel?